code/obj/vehicle.dm

```diff
@@ -2263,7 +2265,7 @@ obj/vehicle/forklift/attackby(var/obj/item/I, var/mob/user)
 
 	//attacking rider on forklift
 	if(rider && rider_visible && I.force)
-		I.attack(rider, user)
+		..()
 		I.visible_message("<span style=\"color:red\">[user] swings at [rider] with [I]!</span>")
```

Comment:
Personally I'd just move the `visible_message` to the parent and then just do `..()` here. Messages (feedback in general) are good for gameplay.

Reply:
Done